packages/components/src/Menu/Menu.tsx

```diff
@@ -71,7 +71,7 @@ const composeOverlayVariation = {
   visible: { opacity: 1 },
 };
 
-const variation = {
+const animationVariation = {
```

Comment:
"variation" was getting in the way of the new prop name

this version is just internal anyway for the framer config